### PR TITLE
Mention typo3-rector in the Upgrade Guide

### PR DESCRIPTION
**Pull request by wolffc**

### CODE REVIEW

File: Documentation/Major/PreupgradeTasks/Deprecations.rst.txt [hunk 1]
```diff
@@ -15,6 +15,9 @@ The extension scanner will show the corresponding changelog which contains
 a description of how to migrate your code. See :ref:`check-the-changelog-and-news-md`
 for more information about the Changelogs and how to read them.
 
+In addition you can use the tool :ref:`typo3-rector` to automatically refactor the code for a lot of deprecations
+(https://github.com/sabbelasichon/typo3-rector)
+
 .. note::
```
Comment (linawolf) [inline Documentation/Major/PreupgradeTasks/Deprecations.rst.txt:21]:
```suggestion
In addition you can use the tool `typo3-rector <https://github.com/sabbelasichon/typo3-rector>`__ to automatically refactor the code for a lot of deprecations.

.. note::
```
User external links here.

**Comment by linawolf** [comment]
Thanks a lot for your contribution, just a small remark

**Comment by wolffc** [comment]
thanks @linawolf 
i have no Restructured Text Knowlage

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
@wolffc We have a cheat sheet for you: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/CheatSheet.html#links

**Pull request merged**